packages/eslint-plugin-pf-codemods/test/rules/v5/toolbar-remove-visiblity.js

```diff
@@ -0,0 +1,24 @@
+const ruleTester = require('../../ruletester');
+const rule = require('../../../lib/rules/v5/toolbar-remove-visiblity');
+
+ruleTester.run("toolbar-remove-visiblity", rule, {
+  valid: [
+    {
+      code: `import { ToolbarContent } from '@patternfly/react-core'; <ToolbarContent visibility />`,
+    },
+    {
+      // No @patternfly/react-core import
+      code: `<ToolbarContent visibility />`,
```

Comment:
I noticed this after... but the nonpf test is actually to test the changes don't affect nonpf components of the same name. I've added the small change to my existing pr https://github.com/patternfly/pf-codemods/pull/174/files#diff-287295441f7c3064650362745a50c5f18fa112fc92dcd0c7e3aca3450ccc6f4dR11